### PR TITLE
Added ability to append items to all logs

### PR DESCRIPTION
**Pull request by mckaymic**

### CODE REVIEW

File: src/bunyan-api.js [hunk 1]
```diff
@@ -18,25 +18,35 @@ const bunyan = require('bunyan');
...
-        name: name || 'MustacheTemplate',
-        streams: [{
-          level: (process.env.LOG_LEVEL || 'info'),
-          stream: process.stdout // log LOG_LEVEL and above to stdout
-        }],
-        serializers: bunyan.stdSerializers
-      });
+      if ( process.env.LOG_APPENDAGE ) {
+        try {
+          Object.assign(loggerConfig, JSON.parse( process.env.LOG_APPENDAGE ) );
```
Comment (adamkingit) [inline src/bunyan-api.js:34]: What format would `process.env.LOG_APPENDAGE`? 
If we really want it to be JSON, we need to parse separately and handle parse errors.

File: src/bunyan-api.js [hunk 2]
```diff
@@ -18,25 +18,35 @@ const bunyan = require('bunyan');
 
 module.exports = {
   'createLogger': function (name) {
+    
+    const loggerConfig = {
+      name: name || 'MustacheTemplate',
```
Comment (adamkingit) [inline src/bunyan-api.js:23]: Consider specifying `MustacheTemplate` as a parameter default

**Review by adamkingit — DISMISSED** [review]
Minimally need to handle JSON parse errors

File: src/bunyan-api.js (hunk 1 above)
Comment (mckaymic) [reply src/bunyan-api.js:34]: Yes, it is JSON.  It is in its own try/catch block.

File: src/bunyan-api.js (hunk 2 above)
Comment (mckaymic) [reply src/bunyan-api.js]: Updated.

**Review by alewitt2 — APPROVED** [review]

**Pull request merged**